### ISSUE TITLE
Super street fighter 3d have graphical glitches 

### ISSUE THREAD

**Issue by Tiganjos** [SachinVin/citra_android#126]
I trying Ssf4 3d on my phone xiaomi mi 6 with snapdragon 835 6 gb and have glitches please can you fix this i need your answer

**Comment by Tiganjos** [comment]
Please dev can you fix this game I don't care if runs slow I just need to see it displaying correctly please dev need your answer this is the best game on 3ds

**Comment by HelloOO7** [comment]
Pics? If it's messed up colors, that's a well known issue. Does software renderer work?

Curious, if it's the best game on 3DS and runs terrible on your phone, why don't you play on your 3DS? This project is maintained for fun, it's not meant for people to actually play games with it (yet).

**Comment by Tiganjos** [comment]
If you don't won't to fix game OK I don't care for this emulator either

**Comment by HelloOO7** [comment]
Why open an issue then? Anyway, an official app's in development, so this repo will possibly be slowly suspended in favour of the official one. It will take months, if not years to get to a state similar to PC, so really, you can't expect everything to work so early.

**Comment by Tiganjos** [comment]
Do you know when the official ap will be available?

**Comment by HelloOO7** [comment]
No-one knows. Citra is not a company, so ETAs can't be given as they mostly
don't exist. And technically, the official app already IS available, but
it's just a blank screen with orange circle right now. You can check the
source at citra-emu/citra/src/android.
